projects/etos_suite_runner/src/etos_suite_runner/lib/suite.py

```diff
@@ -318,19 +318,6 @@ def _download_sub_suite(self, environment: dict) -> dict:
         response.raise_for_status()
         return response.json()
 
-    def _announce(self, header: str, body: str) -> None:
```

Comment:
Why remove announcements in the same review? Can it be done separately? Just in case we need to revert changes.

Reply:
They don't do any harm even if they are still there. I'll just remove them in the future if these changes get reverted.